azext_iot/iothub/_help.py

```diff
@@ -309,3 +309,50 @@ def load_iothub_help():
         type: command
         short-summary: Upload a local file as a device to a pre-configured blob storage container.
     """
+
+    helps["iot edge devices"] = """
+        type: group
+        short-summary: Commands to create and manage IoT Edge devices.
```

Comment:
right now there is only create - is this expected?
Can this summary just say manage (without the create)

Reply:
Correct, this simply adds the ability to create groups of new edge devices on a hub (and at the same time clean the hub if necessary). Can definitely change to manage, felt like create was the star here so hence it's inclusion :)